docs/docs/20-quickstart.md

```diff
@@ -508,52 +510,28 @@ the previous section.
     EOF
     ```
     
-  :::info
-  Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
-  :::
-
-1. Use the CLI to view our `Warehouse` resource:
-
-    ```shell
-    kargo get warehouses --project kargo-demo
-    ```
+      :::info
+      Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
+      :::
 
-    Sample output:
+1. After applying the above configuration, you can view the pipeline you just created using the [Kargo Dashboard](https://localhost:31444/).
 
-    ```shell
-    NAME         SHARD   AGE
-    kargo-demo           13s
-    ```
+   1. Choose your `Project`, <Hlt>kargo-demo</Hlt>, from the list of available `Project`s
+   1. You can find the pipeline you've created in the center of your Project screen.
+   1. The pipeline consists of connected rectangles representing:
+      1. <Hlt>Subscription</Hlt>: Has information about the container image repository that the `Warehouse` is subscribed to.
+      1. <Hlt>kargo-kemo</Hlt>: This represents your `Warehouse`. It will automatically subscribe to the image and should produce `Freight` shortly.
+      1. <Hlt>test</Hlt>, <Hlt>uat</Hlt>, and <Hlt>prod</Hlt>: These are the deployment `Stage` resources.
 
-1. Use the CLI to view our three `Stage` resources:
-
-    ```shell
-    kargo get stages --project kargo-demo
-    ```
-
-    Sample output:
-
-    ```shell
-    NAME   SHARD   CURRENT FREIGHT   HEALTH   PHASE           AGE
-    prod           0/1 Fulfilled              NotApplicable   19s
-    test           0/1 Fulfilled              NotApplicable   19s
-    uat            0/1 Fulfilled              NotApplicable   19s
-    ```
+   ![Kargo-dashboard-screenshot](../static/img/kargo-dashboard-projects.png)
 
 1. After a few seconds, our `Warehouse`, which subscribes to the
    `public.ecr.aws/nginx/nginx` container image, also should have already
-   produced `Freight`:
+   produced `Freight`. You can monitor the status of `Freight` in the various stages directly on the interface, 
+   where it will appear under the respective stages (`test`, `uat`, `prod`).
 
-    ```shell
-    kargo get freight --project kargo-demo
-    ```
-
-    Sample output:
+   The top of the screen displays a Freight Timeline. As your application progresses through the pipeline, the timeline will update to show you the current `Stage` of your `Freight`.
```

Comment:
> The top of the screen displays a Freight Timeline.

I think this has been established and can be removed.

> As your application progresses through the pipeline, the timeline will update to show you the current `Stage` of your `Freight`.

I would save this bit of information for _after_ the first Promotion and reword it to call attention to Freight in the Timeline being color-coded to reflect what Stages are currently using it.